invenio_communities/assets/semantic-ui/js/invenio_communities/requests/requests.js

```diff
@@ -346,27 +348,61 @@ export const RequestsResultsItemTemplate = ({ result, community }) => {
     creatorName =
       result.expanded?.created_by.metadata?.title || createdBy.community;
   }
-
   return (
-    <Item className="community-item">
+    <Item key={community} className="community-item">
```

Comment:
is community an object? if yes in this case:
```suggestion
    <Item key={community.id} className="community-item">
```
its a bit strange to pass the whole object as a key